docs/extensions/dnsrecord.md

```diff
@@ -30,13 +30,6 @@ The kubeconfig handed out to end-users does contain this *external domain name*,
 As not every end-user has an own domain, it is possible for Gardener administrators to configure so-called *default domains*.
 If configured, shoots that do not specify a domain explicitly get an *external domain name* based on a default domain (unless explicitly stated that this shoot should not get an external domain name (`.spec.dns.provider=unmanaged`).
 
-### Ingress Domain Name (Deprecated)
-
-Gardener allows to deploy a `nginx-ingress-controller` into a shoot cluster (deprecated).
-This controller is exposed via a public load balancer (again, either IP or hostname).
-Gardener creates a wildcard DNS record pointing to this load balancer.
-`Ingress` resources can later use this wildcard DNS record to expose underlying applications.
```

Comment:
Why is this deleted? To my knowledge, a wildcard DNS record is still created if the Nginx Addon is enabled.

In addition, there should be another paragraph for the `seed-ingress` record.